src/sql/parts/dashboard/widgets/explorer/explorerWidget.component.ts

```diff
@@ -120,6 +121,10 @@ export class ExplorerWidget extends DashboardWidget implements IDashboardWidget,
 			let currentProfile = this._bootstrap.connectionManagementService.connectionInfo.connectionProfile;
 			this._register(toDisposableSubscription(this._bootstrap.metadataService.databaseNames.subscribe(
 				data => {
+					// Handle the case where there is no metadata service
+					if (types.isUndefinedOrNull(data)) {
```

Comment:
Doesn't change the program at all but you could achieve the same result with less code using one line: `data = data || [];`

Reply:
Done, thanks!